crates/solidity/testing/snapshots/cst_output/Expression/unicode_string_literal/generated/0.7.0.yml

```diff
@@ -6,8 +6,8 @@ Source: >
 Errors: []
 
 Tree:
-  - Expression (Rule): # 0..25 'unicode"This Emoji: 😃"'
-      - PrimaryExpression (Rule): # 0..25 'unicode"This Emoji: 😃"'
-          - StringExpression (Rule): # 0..25 'unicode"This Emoji: 😃"'
-              - _REPEATED (Rule): # 0..25 'unicode"This Emoji: 😃"'
-                  - UnicodeStringLiteral (Token): 'unicode"This Emoji: 😃"' # 0..25
+  - Expression (Rule): # 0..22 'unicode"This Emoji: 😃"'
```

Comment:
I suggest keeping byte offsets in tests. Since we are mainly working in Rust, it makes it easier for someone debugging tests to take a slice and inspect the string.

Reply:
no opinion either way, so ok.